openstudiocore/src/model/ThermalZone.cpp

```diff
@@ -2361,6 +2366,11 @@ namespace detail {
       tz.setZoneControlContaminantController(controllerClone);
     }
 
+    if( auto t_color = renderingColor() ) {
+      auto colorClone = t_color->clone(model).cast<RenderingColor>();
```

Comment:
@jmarrec all objects need to do this in clone right?

Reply:
I guess I should (or at least could) make rendering color a child of ThermalZone. ThermalZone is an HVACComponent, but that's also a ParentObject. I must have got thrown off by that.